client/my-sites/site-settings/wrap-settings-form.jsx

```diff
@@ -110,7 +111,13 @@ const wrapSettingsForm = getFormSettings => SettingsForm => {
 			const { fields, settingsFields, siteId, jetpackSettingsUISupported } = this.props;
 			this.props.removeNotice( 'site-settings-save' );
 
-			this.props.saveSiteSettings( siteId, pick( fields, settingsFields.site ) );
+			let settings = pick( fields, settingsFields.site );
+
+			// Omit `timezone` since this value isn't used to save site settings data.
+			settings = omit( settings, 'timezone_value' );
```

Comment:
I'd prefer a whitelist if possible. Is it not possible to get the correct settings object using pick?